sdk/webpubsub/Microsoft.Azure.Functions.Worker.Extensions.WebPubSub/samples/WebPubSubConnectionInputFunction.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Net;
```

Comment:
If these are intended just as snippets for the README, please move to the standard `tests/samples` path an include the [standard verbiage](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/appconfiguration/Azure.Data.AppConfiguration/tests/samples/README.md) in the README.